components/advocates/MeetTheAdvocates.vue

```diff
@@ -43,27 +50,48 @@ import { Component, Prop } from 'vue-property-decorator'
 import AdvocateCard from '~/components/advocates/AdvocateCard.vue'
 import AppMultiSelect from '~/components/ui/AppMultiSelect.vue'
 import AppFieldset from '~/components/ui/AppFieldset.vue'
-import AppCheckbox from '~/components/ui/AppCheckbox.vue'
 import AppFiltersResultsLayout from '~/components/ui/AppFiltersResultsLayout.vue'
 import InfiniteScroll from '~/components/ui/InfiniteScroll.vue'
+import { ADVOCATES_WORLD_REGION_OPTIONS } from '~/store/modules/advocates.ts'
+import { Advocate } from '~/types/advocate'
```

Comment:
Our code is already written in TS. `*d.ts` are used to provide TS type information about an API that's written in JS. You can read more explanation [here](https://stackoverflow.com/questions/21247278/about-d-ts-in-typescript).

We already have the Advocate type on `'~/store/modules/advocates.ts'`, so please, use it

Reply:
Thanks for pointing that out! 🙂

I also found a couple of videos (https://www.youtube.com/watch?v=s_CZeWuEZ_s, https://www.youtube.com/watch?v=GVUPOvCRmSY) as well as the original documentation (https://www.typescriptlang.org/docs/handbook/declaration-files/by-example.html) very informative.

Regarding the **Advocate type**, I believe the type definition shouldn't live in the `store` directory but rather in the existing `types` folder for more logical reusability outside of the store, so I'll be moving it with my new changes.

Reply:
Ok, I see your point and I think having the types on a separate file under /types could be beneficial. Let's create an issue to do this refactor (you can take it if you want to work on it) to ensure we follow the same format for all types we have, not just the advocates. On this PR, please leave the advocates type where it was

Would you mind to create that issue? Thanks!

Reply:
Agree with @y4izus.
We have to be sure that the changes address the description of the issue and move other changes to a different PR with its own issue.
I agree with you on separating `data-types` from the store files, but I'm not sure that `types` folder is the best location.

Reply:
I agree with both 🙂, it makes more sense to address this on a separate issue: https://github.com/Qiskit/qiskit.org/issues/1133

I'll revert these changes.

Reply:
Done. Please check again @y4izus, @korgan00. 🙂